digitalearthau/coherence.py

```diff
@@ -139,18 +194,80 @@ def _check_ancestors(check_ancestors: bool,
                     s for s in siblings
                     if s != dataset and not s.is_archived and s.type == dataset.type
                 ]
-                if siblings:
+
+                # Do not process/check siblings for nbar/nbart/pq albers products
+                if siblings and (str(dataset.type.name) not in IGNORE_SIBLINGS):
```

Comment:
The `(str(dataset.type.name) not in IGNORE_SIBLINGS)` is redundant, it's already checked on line 185.

Reply:
`--check-siblings` command line option removed as this is handled by `--check-downstream` option and by `dea-duplicates` command.